python-main.js

```diff
@@ -206,7 +206,7 @@ function web_editor(config) {
     function setupFeatureFlags() {
         if(config.flags.blocks) {
             $("#command-blockly").removeClass('hidden');
-            // Add includes 
+            // Add includes
```

Comment:
There is another PR touching this line as well, can we undo this whitespace change to avoid conflicts?